interfaces/rail/README.md

```diff
@@ -0,0 +1,118 @@
+
+- **processSEDs.py**: Build a model of fluxes-redshifts from a set of SED templates in the chosen filters. It is possible to use either the CWW model or the Brown model. By now, CWW SED are chosen in the delight configuration file. The fluxes-redshifts models, one per SED template are written in the file '_fluxredshiftmod.txt'.
+
+- It is important to notice that Filters and SED data are shipped in the installation of Delight. This is probably not the best choice because during its processing, Delight writes the resuls of **processFilters.py** and **processSEDs.py** in those directory, unless Delight is installed for each user (python setup.py install --user).
+
+### creation
+
+- **simulateWithSEDs.py**: In **TUTORIAL MODE** it generates mock dataset inside Delight. These data are the traning and validation samples of fluxes-fluxeserrors-redshift drawing ramdomly the type of the underlying SED template Fluxes-Redshifts model build in **processSEDs.py**. These data are written in the ascii files **'trainingFile'** and **'targetFile'** (which true name is given in the configuration parameter file).
+
+### Learn
+
+- **delightLearn.py**:	Build the Gaussian Process from the training data. The Gaussian process is written in the file **'training_paramFile'**.
+- whether one is running in **TUTORIAL MODE** and **STANDARD MODE**, all the training data are processed in one step.
+
+- Note for template estimation of redshift, there is no learning phase. 
+
+### Estimation	
+
+
+							
+- **templateFitting.py**: The Gaussian Process estimation is also based on latent SED templates. These lated SED template are only usefull for validation dataset to help the GP to estimate the redshift. That is why this module is called from RAIL estimation function (note Learn). Namely it writes the file **'redshiftpdfFileTemp'**.
+
+
+- **delightApply.py**:	Estimate the redshift posterior distribution using Gaussian process and writes its results in file  **'globalPDFs'**. 						
+
+## Other Delight modules
+
+- **calibrateTemplateMixturePriors.py**: This module must be used outside RAIL. It is only tool to check. 
+
+
+## Python modules for interface purposes
+
+
+- **makeConfigParam.py**: it generates Delight configuration file **parametersTest.cfg** from Delight configuration file **delightPZ.yaml**. In the case of RAIL estimation, the validation dataset is split into different chunks. In the later case, there must be is one Delight configuration file per chunk. A Delight configuration file must contain all datafile paths used by Delight.
+
+- **convertDESCcat.py**: Convert DESC PhotoZ dataset into ascii files readable by Delight. For now, only files with hdf5 format are used.
+
+- **getDelightRedshiftEstimation.py**: Retrieve results on PhotoZ estimation of Delight for RAIL. 
+
```

Comment:
if by interface we mean here "running with rail", then this code probably needs to live in RAIL